CODE_OF_CONDUCT.md

```diff
@@ -6,14 +6,31 @@ identity and expression, sexual orientation, disability, personal
 appearance, body size, race, ethnicity, age, religion, nationality, or
 other such characteristics.
 
-Everyone is expected to follow the [Scala Code of Conduct] when
-discussing the project on the available communication channels. If you
-are being harassed, please contact us immediately so that we can
-support you.
+The ZIO project follows the [Scala Code of Conduct](https://www.scala-lang.org/conduct/), with
+adjustments with clarifications with regard to moderation that are detailed below.
+All participants, contributors and members are expected to follow the Scala Code of Conduct
+when discussing the project on the available communication channels. If you are being harassed, 
+please contact us immediately so that we can support you.
 
- ## Moderation
+## Moderation and Steering Committee
+ 
+The ZIO project's group of moderators is the Steering Committee team. The members currently are:
+* Itamar Ravid (@iravid)
+* John De Goes (@jdegoes)
+* Kai (@neko-kai)
+* Paul Shirshov (@pshirshov)
+* Salar Rahmanian (@softinio)
 
-For any questions, concerns, or moderation requests please contact a
-member of the project.
+The ZIO project requires that drastic moderation actions detailed in the code of 
+conduct - for example, removing a user from the Gitter channel - be agreed upon
+a group of over 2/3rds of the steering committee.
```

Comment:
It is kinda awkward to have 2/3 vote in a group of 5 :).

Reply:
I propose adding Wiem to the steering committee.

Reply:
/cc @wi101

Reply:
crazy idea: 100%

Reply:
I hope that 2/3rds of 7-8 members (rounded up this is 5 members) is a large enough majority to not be sufficiently different from 100%, while still allowing for progress in the face of disagreements.